Library/Homebrew/software_spec.rb

```diff
@@ -344,3 +344,17 @@ def checksums
     checksums
   end
 end
+
+class PourBottleCheck
+  def initialize(formula)
+    @formula = formula
+  end
+
+  def reason(reason)
+    @formula.pour_bottle_check_unsatisfied_reason(reason)
```

Comment:
> I think it's better to use attr_accessor rather than attr_rw here.

:ok: 

> Also any chance to have a shorter variable name here?

Given it's not used widely or externally and there was some justified confusion about the existing name I'd rather it was super explicit.